central/main.go

```diff
@@ -826,6 +827,23 @@ func customRoutes() (customRoutes []routes.CustomRoute) {
 		},
 	)
 
+	scannerV4DefinitionsRoute := "/api/extensions/scanner-v4/definitions"
+	utils.CrashOnError(err)
+	customRoutes = append(customRoutes,
+		routes.CustomRoute{
+			Route: scannerV4DefinitionsRoute,
+			Authorizer: perrpc.FromMap(map[authz.Authorizer][]string{
+				or.SensorOr(
+					or.ScannerOr(
+						user.With(permissions.View(resources.Administration)))): {
+					routes.RPCNameForHTTP(scannerV4DefinitionsRoute, http.MethodGet),
+				},
```

Comment:
Am I seeing this wrong, or are we authorizing Sensor to connect, but leaving Scanner out?

Reply:
is `routes.RPCNameForHTTP` wrong? I tried to hit the endpoint with http request and it worked. Do you have any suggestions?

Reply:
I think this means both sensor and scanner?
```
Authorizer: perrpc.FromMap(map[authz.Authorizer][]string{
				or.SensorOr(
					or.ScannerOr(
						user.With(permissions.View(resources.Administration)))): {
					routes.RPCNameForHTTP(scannerV4DefinitionsRoute, http.MethodGet),
				},
			}),
```